cmd/network/status.go

```diff
@@ -3,40 +3,191 @@ package network
 	cliConfig "github.com/oasisprotocol/cli/config"
 )
 
-var statusCmd = &cobra.Command{
-	Use:   "status",
-	Short: "Show the current status of the node and the network",
-	Args:  cobra.NoArgs,
-	Run: func(cmd *cobra.Command, args []string) {
-		cfg := cliConfig.Global()
-		npa := common.GetNPASelection(cfg)
+func getParatimeName(cfg *cliConfig.Config, id string) string {
+	for _, net := range cfg.Networks.All {
+		for ptName, pt := range net.ParaTimes.All {
+			if id == pt.ID {
+				return (ptName)
+			}
+		}
+	}
+	return ("unknown")
+}
+
+const (
+	formatJSON = "json"
+)
 
-		// Establish connection with the target network.
-		ctx := context.Background()
-		conn, err := connection.Connect(ctx, npa.Network)
-		cobra.CheckErr(err)
+var (
+	outputFormat string
 
-		ctrlConn := conn.Control()
+	statusCmd = &cobra.Command{
+		Use:   "status",
+		Short: "Show the current status of the node and the network",
+		Args:  cobra.NoArgs,
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := cliConfig.Global()
+			npa := common.GetNPASelection(cfg)
 
-		nodeStatus, err := ctrlConn.GetStatus(ctx)
-		cobra.CheckErr(err)
+			// Establish connection with the target network.
+			ctx := context.Background()
+			conn, err := connection.Connect(ctx, npa.Network)
+			cobra.CheckErr(err)
 
-		nodeStr, err := common.PrettyJSONMarshal(nodeStatus)
-		cobra.CheckErr(err)
+			ctrlConn := conn.Control()
 
-		fmt.Println(string(nodeStr))
-	},
-}
+			nodeStatus, err := ctrlConn.GetStatus(ctx)
+			cobra.CheckErr(err)
+
+			switch outputFormat {
+			case formatJSON:
+				nodeStr, err := common.PrettyJSONMarshal(nodeStatus)
+				cobra.CheckErr(err)
+
+				fmt.Println(string(nodeStr))
+			default:
+				fmt.Println("=== NETWORK STATUS ===")
+				fmt.Printf("Network:                %s", npa.PrettyPrintNetwork())
+				fmt.Println()
+
+				fmt.Printf("Node's ID:              %s", nodeStatus.Identity.Node)
+				fmt.Println()
+
+				fmt.Printf("Core version:           %s", nodeStatus.SoftwareVersion)
+				fmt.Println()
+
+				fmt.Println()
+				fmt.Printf("==== Consensus ====")
+				fmt.Println()
+
+				consensus := nodeStatus.Consensus
+				if consensus != nil {
+					fmt.Printf("Status:                 %s", consensus.Status.String())
+					fmt.Println()
+
+					fmt.Printf("Version:                %s", consensus.Version.String())
+					fmt.Println()
+
+					fmt.Printf("Chain context:          %s", consensus.ChainContext)
+					fmt.Println()
+
+					date := time.Unix(nodeStatus.Consensus.LatestTime.Unix(), 0)
+					fmt.Printf("Latest height:          %d (%s)", consensus.LatestHeight, date)
+					fmt.Println()
+
+					fmt.Printf("Latest block hash:      %s", consensus.LatestHash)
+					fmt.Println()
+
+					fmt.Printf("Latest epoch:           %d", consensus.LatestEpoch)
+					fmt.Println()
+
+					fmt.Printf("Is validator:           %t", consensus.IsValidator)
+					fmt.Println()
+				}
+
+				registration := nodeStatus.Registration
+				if registration != nil {
```

Comment:
This doesn't seem to be working for me:
```
Registration:           false
  Last:                 0001-01-01 00:00:00 +0000 UTC
  Last attempt:         0001-01-01 00:00:00 +0000 UTC
  Last attempt error:   
```

I think if Last, Last attempt and Last attempt error are zero/nil, then they should be hidden.